RestInfrastructure/Services/UserService.cs

```diff
@@ -29,5 +30,15 @@ public List<UserDto> GetUsers(string sex = "", int olderThan = 0, int yongerThan
 
             return response.Data;
         }
+
+        public List<string> CreateUser(UserDto user, HttpStatusCode expectedHttpStatusCode)
+        {
+            var request = _apiWriteRestClientInstance.CreateRestRequest("http://localhost:49000/users", Method.Post);
```

Comment:
I would create a global variable like BaseURL for http://localhost:49000/

Reply:
Ok, this is particularly implemented. Need just to replace hardcoded value. I will do this in task 70.